### ISSUE TITLE
Ability to add custom Blocked services

### ISSUE THREAD

**Issue by gaetan-p** [AdguardTeam/AdGuardHome#4674]
I think adding the ability to defined our own custom "service" in "Blocked Services" page would be great.

The idea is to have a button "Add custom blocked services"
You define a name, possibly an icon if you want to and then write list the domain names you'd like under this custom "service".

When saved, the new custom services would appear in the list as any other with the ability to block/unblock it at will.

For instance I can add a custom that I name "HBO" and list hbo.com and hbomax.com as the related domain names.
Then I have a button HBO in the Blocked services page that I can block/unblock at will.

**Comment by DandelionSprout** [comment]
Sounds like it'd be a better idea to simply create custom filterlists instead, which'd have the same capabilities and then some.

**Comment by gaetan-p** [comment]
Of course I guess I can do that, but then I don't have a simple button somewhere that I click to enable/disable it whenever I want, or do I ?

**Comment by DandelionSprout** [comment]
If the filterlist was hosted online on GitHub/GitLab/Bitbucket/similar, and then added to the "DNS blocklists" menu, there'd at least be a toggle to turn it on/off for all devices.

**Comment by gaetan-p** [comment]
Not everyone is able or knowledgeable to do so - I am not : ) 
I am looking at it from the lambda user perspective (which I belong to) and the simplicity of it. But maybe it's not the target "audience" for AdGuard Home, I don't know, just wanted to suggest at least.

**Comment by fernvenue** [comment]
> Sounds like it'd be a better idea to simply create custom filterlists instead, which'd have the same capabilities and then some.

@DandelionSprout Actually no, you can choose to enable or disable some different services for each client, but you can't do that with custom filter lists.

**Comment by Aikatsui** [comment]
Duplicate of #1692
